cellprofiler/modules/measureobjectsizeshape.py

```diff
@@ -112,6 +112,8 @@
    angles). The minimum and maximum Feret diameters are the smallest and
    largest possible diameters, rotating the calipers along all possible
    angles.
+-  *BoundingBoxMinX, BoundingBoxMinY, BoundingBoxMaxX, BoundingBoxMaxY:* *(2D only)* The
+   bounding box coordinates for an object. X corresponds to rows and Y corresponds to columns.
```

Comment:
"X corresponds to rows and Y corresponds to columns."- this is vague.  Are you saying it's opposite to what we'd expect (ie X corresponds to position up-and-down and Y corresponds to position left-and-right)?  If so, we should change how the variables are assigned to correspond to the "expected" definitions of X and Y.  If not, we should clarify or omit this sentence.

Reply:
Confirmed that the coordinates are currently backwards relative to meanings of X and Y elsewhere in CP